src/integrations/feed/configs.ts

```diff
@@ -88,6 +98,9 @@ const addFiltersToConfig = ({
     opts.includeBlockedContentCuration
```

Comment:
Should we add `includeAllowedContentCuration` or rename to `includeContentCuration` if we want to keep them together.

Reply:
includeContentCuration should just be the default AFAIK when there is blocked ones.
It's more a retro fix at this stage when you do have blocked items we also pass which types are supposed to be included.

Overall don't see a need to have it optional now, but maybe miss some potential context here.

Reply:
Options like `opts. includeBlockedContentCuration` can be passed from feed generators to avoid sending some filtering options to feed request. 

I think we should rename to `includeContentCuration` then so we keep them together.

Reply:
My main concern is that it's already passed in multiple places now (*and maybe even LOFN not sure)
This way we keep all logic and at very end send this extra part.

I do get your comment, but feels more dangerous to do now.
I'll check as well with @idoshamun if maybe I missed anything.

Reply:
@capJavert Decided to rename anyway :)